raster-foundry/raster-foundry-api-spec
# Api spec audit part 1

## Overview
Audit API spec and correct any differences:
  - [x] get `projects/` + related objects / parameters
  - [x] post `projects/` + related objects
  - [x] get `projects/{id}` + related objects / parameters
  - [x] put `projects/{id}` + related objects
  - [x] delete `proejcts/{id}`
  - [x] post `projects/{id}/project-color-mode` + related objects / parameters
  - [x] get `projects/{id}/labels`
  - [x] get `projects/{id}/annotation-groups`
  - [x] post `projects/{id}/annotation-groups`
  - [x] get `projects/{id}/annotation-groups/summary`

### Checklist


### Demo
Swagger editor is not complaining
![image](https://user-images.githubusercontent.com/4392704/48375311-8a52d500-e695-11e8-979f-24dd7e45aa0c.png)


## Testing Instructions
 * Verify that the changes to the spec match the actual api
 * Verify that the python client correctly serialized the changes

Closes https://github.com/raster-foundry/raster-foundry-api-spec/issues/65
